### PR TITLE
The card profile is performed

### PR DESCRIPTION
**Pull request by Elvis-Mieles**
##summary
The card profile is performed

### CODE REVIEW

File: src/test/java/training/profile/updateprofile.feature [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Feature:  Update profile
+
+  Background:
+    * url url
+
+  @flow
+  Scenario Outline:  Update profile
```
Comment (jmzagowolox) [inline src/test/java/training/profile/updateprofile.feature:7]: if you only gonna do an example, don't use outline scenarios

File: src/test/java/training/profile/updateprofile.feature [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    And assert response.user.username == '<username>'
+    And assert response.user.email == '<email>'
+    And assert response.user.bio == '<bio>'
+    And assert response.user.image == '<imagenUrl>'
+    And def getProfile = call read('getprofile.feature@getprofile')
+    And assert getProfile.response.user.username === response.user.'<username>'
+
+    Examples:
+    |username        |email                     |bio           |imagenUrl          |statusCode|
+    |migueyliuiyles  |migueeysiuihf@gmail.com   |soy estudiante|imagen.png         |200       |
```
Comment (jmzagowolox) [inline src/test/java/training/profile/updateprofile.feature]: can you change every fields? is there any other validation to do?

File: src/test/java/training/profile/getprofile.feature [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  Scenario:  get profile
+
+    * def responseGet = read('training/profile/schemegetprofile.json')
+
+    Given path 'user'
+    And header Authorization = token
+    When method get
+    Then status 200
+    And assert response.user.username == username
+    And match  response == responseGet
```
Comment (jmzagowolox) [inline src/test/java/training/profile/getprofile.feature:15]: must the token match with the value on vaariable (`token`)?

File: src/test/java/training/profile/schemegetprofile.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "user": {
+    "id": "#number",
+    "email": "#string",
+    "username": "#string",
+    "bio": "#string",
+    "image": "##Sstring",
```
Comment (jmzagowolox) [inline src/test/java/training/profile/schemegetprofile.json]: check this typo

File: src/test/java/training/profile/updateprofile.feature [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    And request updateProfile
+    When method put
+    Then status <statusCode>
+    And match  response == profileSuccessfully
+    And assert response.user.username == '<username>'
+    And assert response.user.email == '<email>'
+    And assert response.user.bio == '<bio>'
+    And assert response.user.image == '<imagenUrl>'
+    And def getProfile = call read('getprofile.feature@getprofile')
+    And assert getProfile.response.user.username === response.user.'<username>'
```
Comment (jmzagowolox) [inline src/test/java/training/profile/updateprofile.feature]: is that ok? is it work? why do you do that?

File: src/test/java/training/profile/schemegetprofile.json [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (jmzagowolox) [inline src/test/java/training/profile/schemegetprofile.json]: remember the name of the schemAs must be representative

**Review by jmzagowolox — CHANGES_REQUESTED** [review]

**Comment by Elvis-Mieles** [comment]
##summary
The @unitTest scenario has been added in the updateprofile feature

File: src/test/java/training/profile/updateprofile.feature [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+  Scenario Outline:  Update wrong profile
+    * def updateProfile = {"user":{"username":'<username>',"email":'<email>',"bio":'<bio>',"image":'<imagenUrl>'}}
+    * def profileIncorrect = read('training/profile/userincorrectschema.json')
+
+    Given path 'user'
+    And header Authorization = token
+    And request updateProfile
+    When method put
+    Then status <statusCode>
+    And match  response == profileIncorrect
```
Comment (jmzagowolox) [inline src/test/java/training/profile/updateprofile.feature:38]: you always match with the schema what contains the "username" attribute, but you have an example where the email is incorrect. Is that error returned inside the username attribute?

**Review by jmzagowolox — CHANGES_REQUESTED** [review]

**Comment by Elvis-Mieles** [comment]
##summary
The wrong user schema.json file is modified

File: src/test/java/training/profile/userincorrectschema.json [hunk 8]
```diff
@@ -0,0 +1,8 @@
+{
+  "errors": {
+    "#string": [
+      "##string",
+      "##string"
```
Comment (jmzagowolox) [inline src/test/java/training/profile/userincorrectschema.json]: "#string": "#[] #string"

File: src/test/java/training/profile/userscheme.json [hunk 9]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (jmzagowolox) [inline src/test/java/training/profile/userscheme.json]: the correct word is SCHEMA

**Review by jmzagowolox — CHANGES_REQUESTED** [review]

**Comment by Elvis-Mieles** [comment]
##summary
The wrong userschema.json file is modified

**Review by jmzagowolox — APPROVED** [review]

**Pull request merged**